providers/keycloak_oidc_test.go

```diff
@@ -32,11 +33,20 @@ func getAccessToken() string {
 }
 
 func newTestKeycloakOIDCSetup() (*httptest.Server, *KeycloakOIDCProvider) {
-	redeemURL, server := newOIDCServer([]byte(fmt.Sprintf(`{"email": "new@thing.com", "expires_in": 300, "access_token": "%v"}`, getAccessToken())))
+	redeemURL, server := newKeycloakOIDCServer([]byte(fmt.Sprintf(`{"email": "new@thing.com", "expires_in": 300, "access_token": "%v"}`, getAccessToken())))
```

Comment:
Is this feature being implemented for Keycloak or is this more generally available across OIDC providers?

Reply:
Its more generic. We have this feature on our own identity provider internally.